internal/resources/grafana/resource_organization_preferences.go

```diff
@@ -62,11 +62,12 @@ func ResourceOrganizationPreferences() *schema.Resource {
 				Description:  "The Organization timezone. Available values are `utc`, `browser`, or an empty string for the default.",
 				ValidateFunc: validation.StringInSlice([]string{"utc", "browser", ""}, false),
 			},
-			// TODO: add validation?
 			"week_start": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "The Organization week start.",
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "The Organization week start day. Available values are `sunday`, `monday`, `saturday`, or an empty string for the default.",
+				ValidateFunc: validation.StringInSlice([]string{"sunday", "monday", "saturday", ""}, false),
```

Comment:
```suggestion
				ValidateFunc: validation.StringInSlice([]string{"sunday", "monday", "saturday", ""}, true),
```

Should it be case-insensitive?

```
=== CONT  TestAccResourceOrganizationPreferences_WithDashboardID
    resource_organization_preferences_test.go:20: Step 1/5 error: Error running pre-apply refresh: exit status 1
        
        Error: expected week_start to be one of ["sunday" "monday" "saturday" ""], got Monday
        
          with grafana_organization_preferences.test,
          on terraform_plugin_test.tf line 18, in resource "grafana_organization_preferences" "test":
          18:   week_start = "Monday"
```

Reply:
Compared with Grafana's source code, the name of the day is fully case-sensitive no ? 

https://github.com/grafana/grafana/blob/main/pkg/kinds/dashboard/dashboard_spec_gen.go#L786

Reply:
@julienduchesne  ?

Reply:
You are right that the lowercase value is right and uppercase isn't. I tried `Monday` and it just doesn't work. `monday` does. We do have some invalid examples and tests that need to be changed though. Ex: https://github.com/grafana/terraform-provider-grafana/blob/master/examples/resources/grafana_organization_preferences/resource.tf#L4

Reply:
I will modify the incorrect tests.